django_force_logout/middleware.py

```diff
@@ -34,10 +35,18 @@ def process_request(self, request):
                 request.session[self.SESSION_KEY],
             )
         except KeyError:
-            # May not have logged in since we started populating this key.
-            return
-
-        if timestamp > user_timestamp:
-            return
+            # May not have logged in since we started populating this key.  If
+            # so, and the logout timestamp is set, we're supposed to log out
+            # this user.
+            pass
```

Comment:
Can we invert the logic here so we don't have an ugly ``if X: pass else Y``? Seems rather un-Pythonic